features-support/step_definitions/parsing.js

```diff
@@ -12,25 +12,43 @@ module.exports = function() {
   var visitor;
   var features;
 
-  this.Given(/^the feature file$/, function (string) {
+  this.Given(/^the feature file\.?$/, function (string, done) {
     featureText = string;
+    done();
```

Comment:
Good question. So I thought it was necessary on the final step in order for Cucumber to correctly end the stream. Turns out that is only true is the step function takes a parameter in the position where the callback would be passed for that step because it checks for the existence of the parameter

https://github.com/cucumber/cucumber-js/blob/master/lib/cucumber/support_code/step_definition.js#L95

So I'll delete the callbacks.